### DISCUSSION TITLE
Explore the use of https://dozerdb.org/

### DISCUSSION THREAD

**Discussion by alexgarel** [openfoodfacts/taxonomy-editor#255] [Ideas]
https://dozerdb.org/ brings multiple database to neo4j community edition.

This could be a good thing instead of having one database with all graphs.

**Comment by jmsuhy** [comment]
Feel free to reach out to me if you have any questions.  I created DozerDB and would love to hear your feedback on the roadmap for this year.

My email is jmsuhy@greystonesgroup.com
